exasol_integration_test_docker_environment/lib/api/common.py

```diff
@@ -105,8 +104,8 @@ def import_build_steps(flavor_path: Tuple[str, ...]):
         path_to_build_steps = Path(path).joinpath("flavor_base/build_steps.py")
         module_name_for_build_steps = extract_modulename_for_build_steps(path)
         spec = importlib.util.spec_from_file_location(module_name_for_build_steps, path_to_build_steps)
-        module = importlib.util.module_from_spec(spec)
-        spec.loader.exec_module(module)
+        module = importlib.util.module_from_spec(spec) # type: ignore
```

Comment:
why does it complain here?

Reply:
```
exasol_integration_test_docker_environment/lib/api/common.py:107:50: error: Argument 1 to "module_from_spec" has incompatible type "ModuleSpec | None"; expected "ModuleSpec"  [arg-type]
            module = importlib.util.module_from_spec(spec)
```

I also could fix it using a simple assert, like I did at other places

```
        assert spec and spec.loader
        module = importlib.util.module_from_spec(spec)
        spec.loader.exec_module(module)
```

Reply:
Going with the assertion